lightning/src/blinded_path/mod.rs

```diff
@@ -76,6 +76,16 @@ impl BlindedPath {
 		})
 	}
 
+	/// Create a one-hop blinded path for a payment.
+	pub fn one_hop_for_payment<ES: EntropySource, T: secp256k1::Signing + secp256k1::Verification>(
+		payee_node_id: PublicKey, payee_tlvs: payment::ReceiveTlvs, entropy_source: &ES,
+		secp_ctx: &Secp256k1<T>
+	) -> Result<(BlindedPayInfo, Self), ()> {
+		Self::new_for_payment(
+			&[], payee_node_id, payee_tlvs, u64::max_value(), entropy_source, secp_ctx
```

Comment:
Does the particular `htlc_maximum_msat` value here matter?

Reply:
No, it won't be considered in pathfinding. Added a comment.